tests/foreman/api/test_syncplan.py

```diff
@@ -221,17 +257,13 @@ def test_update_sync_plan_with_random_name(self, name):
         @Feature: SyncPlan
 
         """
+        syncplan = entities.SyncPlan(
```

Comment:
Can't - resulting line will be 86 chars long. But thanks for pointing, will double check everything and merge into one line where it's possible.